src/methods/intersects.jl

```diff
@@ -1,6 +1,6 @@
 # # Intersection checks
 
-export intersects, intersection, intersection_points
+export intersects, intersection, intersection_points, union, difference
```

Comment:
I think we no longer export intersection, union, and difference

Reply:
I only moved the intersection of two polygons code to a new file. The intersection of other types of geometries is still in `intersects.jl`. In this case, should I still export intersection from `intersects.jl`? Or should I moved the intersection of other geometries code into `intersection.jl`?

Reply:
Oh! Let's move the others as well. Good catch. I forgot that any of those even existed yet